### PR TITLE
Fix ConditionalRouterIntegrationTest

### PR DESCRIPTION
**Pull request by prabushi**
## Description
Fixed ConditionalRouterIntegrationTest test cases by correcting the synapse artifact loading of dynamic_seq1 and ConditionalRouterIntegrationTestProxy.

### CODE REVIEW

**Comment by milindaperera** [comment]
Let's clean PR description

File: integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-  ~  Copyright (c) 2005-2010, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+  ~  Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (milindaperera) [inline integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml]: We should not update the license if it is a existing file

Comment (prabushi) [reply integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml]: This was moved from tests-mediator-1/src/test/resources/artifacts/ESB/synapseconfig/filters/conditional_router/dynamic_seq1.xml to tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml.

**Comment by prabushi** [comment]
> Let's clean PR description

Updated.

File: integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml (hunk 1 above)
Comment (milindaperera) [reply integration/mediation-tests/tests-mediator-1/src/test/resources/artifacts/ESB/server/registry/config/filters/dynamic_seq1.xml]: but still the same file with original content just restructuring.

**Pull request merged**